sleuth_crawler/scraper/scraper/spiders/parsers/generic_page_parser.py

```diff
@@ -0,0 +1,22 @@
+import scrapy
+import sleuth_crawler.scraper.scraper.spiders.parsers.utils as utils
+from sleuth_crawler.scraper.scraper.items import ScrapyGenericPage
+
+def parse_generic_item(response, extractor):
+    """
+    Scrape generic page
+    """
+    title = utils.extract_element(response.xpath("//title/text()"), 0)
+    desc = utils.extract_element(response.xpath("//meta[@name='description']/@content"), 0)
```

Comment:
You might want to set a default value for `title` and `desc` here, since `extract_element` might return `None`, and we probably don't want to have that floating around.

Reply:
fixed in 779ea0c